src/asset_folder_importer/database.py

```diff
@@ -165,6 +176,10 @@ def check_schema_22(self):
             self.update_schema_22()
             self.conn.commit()
 
+    def check_schema_23(self):
+        if self._has_table('files'):
+            self.update_schema_23()
```

Comment:
this will run the entire schema update (rebuilding the indices) on every startup, we don't want that.  Need to either build a check to see if the update has been applied, or when you apply it stick a value into the sysparam table that can be checked

Reply:
As you suggested, I have re written it so it tests of an old constraint existing and if it finds it removes the old constraints and installs new ones with new names.